app/nypr-m-newsletter-form.js

```diff
@@ -0,0 +1,123 @@
+// BEGIN-SNIPPET nypr-m-newsletter-form.js
+import Component from '@ember/component';
+import layout from '../templates/components/nypr-m-newsletter-form';
+
+import { computed } from '@ember/object';
+import { assert } from '@ember/debug';
+import { task } from 'ember-concurrency';
+
+import fetch from 'fetch';
+
+export const DEFAULT_LEGAL = "By submitting y=our information, you're agreeing to receive communications from New York Public Radio in accordance with our Terms."
```

Comment:
Delete `app/nypr-m-newsletter-form.js` from this PR and use the `nypr-design-system` version, because it looks like the only change to this file from the `nypr-design-system` version is a typo:

```
-export const DEFAULT_LEGAL = "By submitting y=our information, you're agreeing to receive communications from New York Public Radio in accordance with our Terms."
+export const DEFAULT_LEGAL = "By submitting your information, you're agreeing to receive communications from New York Public Radio in accordance with our Terms."
```

Reply:
You should be able to delete the entire `app/nypr-m-newsletter-form.js` from this PR.